stubs/ujson/ujson.pyi

```diff
@@ -47,6 +47,6 @@ def dump(
 ) -> None: ...
 def decode(s: AnyStr, precise_float: bool = ...) -> Any: ...
```

Comment:
@AlexWaygood should I also remove these (since they aren't being used as a TypeVar)?

Reply:
Yes, they look incorrect. I didn't want to ask you to change them in this PR since they are unrelated to the change you were making, but a followup PR fixing these would be welcome! :)